en/drafts/originals/analyzing-multilingual-text-nltk-spacy-stanza.md

```diff
@@ -852,11 +821,11 @@ word: .	upos: PUNCT
 
 ### Lemmatization
 
-Finally, let's perform lemmatization on our sentences using spaCy and Stanza (NLTK does not provide out-of-the-box lemmatization for non-English languages). Lemmatization is the process of grouping together the inflected forms of a word so they can be analysed as a single item, identified by the word's lemma, or dictionary form. The word "typed," for example, would be reduced to the lemma "type," removing the "-d" suffix to return the word to a more basic form.
+Finally, let's perform lemmatization on our sentences using spaCy and Stanza (NLTK does not provide out-of-the-box lemmatization for non-English languages). Lemmatization is the process of grouping together the [inflected](https://en.wikipedia.org/wiki/Inflection) forms of a word so they can be analysed as a single item, the 'lemma'. The word _typed_, for example, would be reduced to the lemma _type_, removing the '-d' suffix to simplify the word into a basic form.
```

Comment:
I don't fully understand the sentence which says: `Lemmatization is the process of grouping together the inflected forms of a word so they can be analysed as a single item, the 'lemma'`. Do you mean grouping the inflected forms with the lemmatized form? Could you perhaps rephrase these couple sentences for clarity?

Reply:
Hi @ian-nai,

I think I'm still a bit unsure about the use of the term "grouped". How about:

Lemmatization is the process of replacing all the [inflected](https://en.wikipedia.org/wiki/Inflection) forms of a word (e.g., _looks_, _looked_) with a single item, the 'lemma', which is the word in its simple form (in this case, _look_).  For example, the sentence _I typed a letter_ would, if lemmatized, be changed to _I type a letter_. The word _typed_ was reduced to the lemma _type_, removing the '-d' suffix to simplify the word into a basic form.